Source/SharpNav/Crowds/PathCorridor.cs

```diff
@@ -270,7 +270,16 @@ public int MergeCorridorStartMoved(Path path, List<NavPolyId> visited)
 
 			//store visited
 			for (int i = 0; i < req; i++)
-				path[i] = visited[(visited.Count - 1) - i];
+            {
```

Comment:
Same here, whitespace of these new lines does not match the spacing of the rest of the file.